### PR TITLE
Contest page class changes

### PR DESCRIPTION
**Pull request by MadjarovaML**

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submission/ProblemSubmission.tsx [hunk 1]
```diff
@@ -92,20 +92,35 @@ const ProblemSubmission = ({ submission }: ISubmissionResultProps) => {
...
     );
 
+    const labelClass = 'label';
+    const labelClassName = concatClassNames(
+        styles.labelContainer,
+        labelClass,
+    );
+
+    const showSubmissionDetailsButtonClassName = 'submissiondetailsButton';
+    const currentSubmissionTypeClass = 'currentSubmissionType';
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submission/ProblemSubmission.tsx:102]: Inconsistency. One has suffix `className` and the other `class`

File: automated_testing/test/tests/auth-tests.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import IndexPage from '../pageobjects/index-page';
 import AuthPage from '../pageobjects/auth-page';
 
-describe('Testing AuthPage', () => {
+describe.skip('Testing AuthPage', () => {
```
Comment (Minkov) [inline automated_testing/test/tests/auth-tests.ts:4]: Should this be in the PR?

File: automated_testing/test/tests/contest-test.ts [hunk 3]
```diff
@@ -14,15 +15,38 @@ describe('Testing contest', () => {
...
+
+    const contestChecker = {
+        leftSideNavigationCheck: (problems) => problems.length > 0,
+        executionTypeCheck: (strategies) => strategies.length > 0,
+    };
+
     before(async () => {
-        await loginWithCredentials(validUserCredentials);
+        await AuthPage.open();
+        await loginInWithValidCredentials();
```
Comment (Minkov) [inline automated_testing/test/tests/contest-test.ts:27]: Nice! Well done!

File: automated_testing/test/tests/index-test.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { beforeEach } from 'mocha';
 import IndexPage from '../pageobjects/index-page';
 
-describe('Testing index', () => {
+describe.skip('Testing index', () => {
```
Comment (Minkov) [inline automated_testing/test/tests/index-test.ts:4]: Is `.skip` for the PR?

**Comment by achobanov** [comment]
WooHoo, Mila code!

**Review by Minkov — DISMISSED** [review]

**Pull request merged**